tests/unit/ConfirmDissolution.spec.ts

```diff
@@ -0,0 +1,40 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { shallowMount } from '@vue/test-utils'
+import { getVuexStore } from '@/store'
+import { ConfirmDissolution } from '@/components/dialogs'
+import { ConfigJson } from '@/resources'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore() as any // remove typings for unit tests
+
+describe('ConfirmDissolution - Displays Confirmation messages', () => {
+  beforeAll(() => {
+    store.state.entityType = 'SP' // now added only for SP
```

Comment:
In a future commit, you could iterate over entity types to test all variations.

Example: https://github.com/bcgov/business-create-ui/blob/main/tests/unit/AgreementType.spec.ts